Release notes: Can't choose UI language anymore

unho opened the pull request:
Fixes #4221.

unho commented:
@translate/core Any quick review would be good since this is some change we want it to make in for next release.

julen commented:
Looks good to me. You may want to add that it's being considered to bring the picker back and link it to #4230.

unho commented:
Added reference to #4230, did some rewording as suggested by @phlax and moved removal of the ability to "Recoverable Features" section on 2.7.0 release notes. I am merging this now.
